catalogs/catalog-jdbc-common/src/main/java/org/apache/gravitino/catalog/jdbc/config/JdbcConfig.java

```diff
@@ -33,7 +34,7 @@ public class JdbcConfig extends Config {
           .doc("The url of the Jdbc connection")
           .version(ConfigConstants.VERSION_0_3_0)
           .stringConf()
-          .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
+          .checkValue(TypeValid::isJdbcURL, ConfigConstants.JDBC_URL_INVALID_ERROR_MSG)
```

Comment:
Could you please also modify other places like JDBC backends in the Iceberg catalog?

Reply:
Hi @yuqi1129 
I have added the uri check for IceBerg catalog and "gravitino.entity.store.relational.jdbcUrl"

Thanks
Qiang

Reply:
Please review the changes carefully. The backend for Iceberg can be Hive or JDBC, so the proposed changes may not apply to the Hive backend. Kindly double-check this. If the valid changes cannot be applied to the Iceberg backend URI, please leave it as is.

Reply:
Also, you can execute `./gradlew build -x web -x test` locally to exclude some code format problems locally to improve development efficiency.